app/code/FrontCommerce/ChronorelaisHeadless/Model/RelayPoint.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace FrontCommerce\ChronorelaisHeadless\Model;
+
+use Chronopost\Chronorelais\Helper\Webservice as HelperWebservice;
+use Exception;
+use Magento\Quote\Model\Quote\AddressFactory;
+
+class RelayPoint
+{
+    protected AddressFactory $addressFactory;
+    protected HelperWebservice $helperWebservice;
+
+    public function __construct(
+        AddressFactory   $addressFactory,
+        HelperWebservice $helperWebservice
+    )
+    {
+        $this->addressFactory = $addressFactory;
+        $this->helperWebservice = $helperWebservice;
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function getRelayPoints(string $methodCode, string $postCode, array $shippingAddress): array
+    {
+        $shippingAddress = $this->addressFactory->create()->setData($shippingAddress);
+
+        return $this->helperWebservice->getPointRelaisByAddress(
+            $methodCode,
+            $shippingAddress
+                ->setData('postcode', $postCode)
+                ->setData('city', $shippingAddress['city'] ?: 'unknown')
+                ->setData('country', $shippingAddress['country_id'] ?: 'unknown')
+                ->setData('street', $shippingAddress['street'] ?: 'unknown')
```

Comment:
and there as well, the post code parameter is supposed to be in lowercase, the other parameters are not supposed to be called `shippingAddress[whatever]` but directly `whatever`.

:warning: `street` is supposed to be array because the customer address might have several lines.

In a nutshell, please make sure the URL in the ticket work ie:

```
# minimum parameters
GET /rest/default/V1/chronorelais/relais?country_id=<COUNTRYCODE>&postcode=<ZIPCODE>
# should work as well
GET /rest/default/V1/chronorelais/relais?country_id=<COUNTRYCODE>&postcode=<ZIPCODE>&city=<CITY>
# complete parameter list
GET /rest/default/V1/chronorelais/relais?country_id=<COUNTRYCODE>&street[]=<STREETVALUE>&postcode=<ZIPCODE>&city=<CITY>
# with several line in the street
GET /rest/default/V1/chronorelais/relais?country_id=<COUNTRYCODE>&street[]=<STREETVALUE1>&street[]=<STREETVALUE2>&postcode=<ZIPCODE>&city=<CITY>
```